### PR TITLE
added mail.ru

### PR DESCRIPTION
**Pull request by hooddy**

### CODE REVIEW

**Comment by hooddy** [comment]
guys i don't get where im rong...

File: _data/sites.json [hunk 1]
```diff
@@ -6871,6 +6871,17 @@
         ]
     },
 
+    {
+        "name": "mail.ru",
```
[the remark this answers is not in the document]
Comment (hooddy) [reply _data/sites.json:6875]: fixed

File: _data/sites.json [hunk 2]
```diff
@@ -6871,6 +6871,17 @@
         ]
     },
 
+    {
+        "name": "Mail.ru",
+        "url": "https://account.mail.ru/user/delete",
+        "difficulty": "easy",
+        "notes_ru": "Войдите в аккаунт, который хотите удалить. Перейдите к форме удаления. Введите имя ящика и пароль. Укажите причину удаления, пароль и код с картинки. Нажмите кнопку \"Удалить\".",
+        "notes": "Login into your account. Go to the deletion form. Enter the mailbox name and password. Specify the reason for the deletion, your accounts password and the captcha code. Click the \"Delete\" button.",
```
Comment (tupaschoal) [inline _data/sites.json:6879]:
```suggestion
        "notes": "Login into your account. Go to the deletion form. Enter the mailbox name and password. Specify the reason for the deletion, your account's password and the captcha code. Click the \"Delete\" button.",
```

**Comment by tupaschoal** [comment]
Thanks for your contribution!

**Pull request merged**